src/software/ai/hl/stp/tactic/goalie/goalie_fsm.cpp

```diff
@@ -182,6 +183,55 @@ void GoalieFSM::panic(const Update &event)
     Angle goalie_orientation =
         (event.common.world_ptr->ball().position() - goalie_pos).orientation();
 
+    Point save_pos = goalie_pos;
+    // First we are trying to determine how much time do we have to reach the ball
+    Duration ball_intercept_time = Duration::fromSeconds(
+        (event.common.world_ptr->ball().position() - save_pos).length() /
+        (std::max(0.1, event.common.world_ptr->ball().velocity().length())));
```

Comment:
```suggestion
        (std::max(std::numeric_limits<double>::epsilon, event.common.world_ptr->ball().velocity().length())));
```
epsilon is a very small non-zero number. this isn't the exact use case to use it per se, but I've seen it being used to avoid a divide-by-zero in a previous coop and I think it looks pretty clean